dd-java-agent/instrumentation/vertx/vertx.gradle

```diff
@@ -1,38 +1,79 @@
+// Modified by SignalFx
 // Set properties before any plugins get loaded
 ext {
   minJavaVersionForTests = JavaVersion.VERSION_1_8
 }
 
 apply from: "${rootDir}/gradle/java.gradle"
+apply from: "${rootDir}/gradle/test-with-kotlin.gradle"
 
 muzzle {
   pass {
     group = "io.vertx"
     module = "vertx-web"
-    versions = "[4.1.0.Final,)"
-    assertInverse = true
-  }
-  pass {
-    group = "io.netty"
-    module = "netty"
-    versions = "[4.1.0.Final,)"
+    versions = "[3.0.0-milestone6, )"
```

Comment:
We're claiming support for 3.1+, any particular reason why this is 3.0.0-m6 ?

Reply:
3.0.0-m6 has the `RoutingContext`, which we instrument. In this version, it is in the `io/vertx/ext/web/` directory, so it passes the muzzle tests. I've updated the support documentation to 3.0.0+ (which is the major release version that the 3.0.0-m6 led up to).